cfme/networks/provider/__init__.py

```diff
@@ -96,7 +96,11 @@ def routers(self):
     def security_groups(self):
         return self.collections.security_groups
 
-    def create(self, cancel=False, validate_credentials=True, validate_inventory=False):
+    def create(self, cancel=False, validate_credentials=True, validate_inventory=False,
+               check_existing=False):
+        if check_existing and self.exists:
```

Comment:
@mshriver please take a look here - i believe its conceptually ok but i wonder if the flag should be called `exists_ok` or if it should be a separate method

Reply:
This was already done in some other PR now.